qipr_approver/approver/workflows/project_crud.py

```diff
@@ -139,6 +146,9 @@ def current_user_can_perform_project_delete(current_user,project):
     user is the owner for the project and the project is editable.
     """
     toast_message = ""
+    if current_user.is_superuser:
+        project.delete(current_user)
+        return 'Deleted Project'
     if(toast_message == "" and project is None):
```

Comment:
An admin should be able to delete a project whether it has been approved or not, line 157